docs/BreakingChanges/065 Corner case WPF bug in which databound text box fails is fixed.md

```diff
@@ -0,0 +1,22 @@
+## 65: Corner case WPF bug in which databound text box fails to update is fixed
+
+### Scope
+Transparent
+
+### Version Introduced
+4.5
+
+### Change Description
+TODO
+
+- [ ] Quirked
+- [ ] Build-time break
+- [ ] Source analyzer planned
+
+### Recommended Action
+TODO
+
+### Affected APIs
+* Not detectable via API analysis
+
+[More information](https://social.msdn.microsoft.com/Forums/vstudio/en-US/c05a8c02-de67-47a9-b4ed-fd8b622a7e4a/if-i-target-net-40-but-run-on-a-machine-that-has-net-45-will-net-40-wpf-bugs-still-be-there?forum=wpf)
```

Comment:
This link references the break, but also is a forum post with tons of unrelated 4.5 compat complaining. We should see if we can find a more targeted/useful link.